components/ClaimSection.tsx

```diff
@@ -13,6 +13,23 @@ export const ClaimSection: React.FC<ClaimSectionProps> = ({
   statusContent,
   detailsContent,
 }) => {
+  let claimDetails: JSX.Element
+
+  if (detailsContent) {
+    claimDetails = (
+      <ClaimDetails
+        loading={loading}
+        programType={detailsContent.programType}
+        benefitYear={detailsContent.benefitYear}
+        claimBalance={detailsContent.claimBalance}
+        weeklyBenefitAmount={detailsContent.weeklyBenefitAmount}
+        lastPaymentIssued={detailsContent.lastPaymentIssued}
+        extensionType={detailsContent.extensionType}
+      />
+    )
+  } else {
+    claimDetails = <div className="claim-details claim-section container" />
```

Comment:
Why do we need to have this empty div?

Reply:
That supports the website layout retaining its proper spacing - specifically the `claim-section` bit